frontend/test/metabase/lib/expressions/resolver.unit.spec.js

```diff
@@ -159,6 +159,44 @@ describe("metabase-lib/expressions/resolve", () => {
     it("should honor CONCAT's implicit casting", () => {
       expect(() => expr(["concat", ["coalesce", "B", 1]])).not.toThrow();
     });
+
+    describe("datetime commands", () => {
+      it("should resolve unchained commmands", () => {
```

Comment:
I would refrain from using `commands` in all these tests to avoid any potential confusions, as the existing ones refer them as `functions`.